payments/billing-sdk/src/main/java/com/appcoins/billing/sdk/billing_support/BillingSupportErrorMapper.kt

```diff
@@ -0,0 +1,26 @@
+package com.appcoins.billing.sdk.billing_support
+
+import com.appcoins.billing.sdk.BillingSdkConstants
+import retrofit2.HttpException
+import javax.inject.Inject
+import javax.inject.Singleton
+
+internal interface BillingSupportErrorMapper {
+  fun mapBillingSupportError(exception: Throwable): Int
+}
+
+@Singleton
+internal class BillingSupportErrorMapperImpl @Inject constructor() : BillingSupportErrorMapper {
+
+  override fun mapBillingSupportError(exception: Throwable): Int {
```

Comment:
In the wallet, in case the exception is an UnknownHostException, they send the error 'RESULT_SERVICE_UNAVAILABLE'. Although maybe it should be IOException that should be checked instead